packages/block-library/src/navigation/index.php

```diff
@@ -234,7 +239,7 @@ function build_navigation_html( $attributes, $block, $colors, $font_sizes, $is_l
 		$html .= '</span>';
 
 		// Append submenu icon to top-level item.
-		if ( $attributes['showSubmenuIcon'] && $is_level_zero && $has_submenu ) {
+		if ( ! empty( $attributes['showSubmenuIcon'] ) && $is_level_zero && $has_submenu ) {
```

Comment:
What relevance does this change have in the original goal of using the filter instead of a new argument?

Reply:
I am not sure why, but in my local setup while testing, I got a notice of this not being defined.